File: Java/DayOne/DayOne.java
import java.util.Scanner;
public class DayOne {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

int countCorrect = 0;
for(int count = 0; count < 10; count++) {
	Double firstNumber = Math.floor(Math.random()  * 100);
	Double secondNumber = Math.floor(Math.random() * 100);
	Double answer = firstNumber - secondNumber;
	
	System.out.print("What is " + firstNumber + " minus " + secondNumber + ": ");
	int userInput = input.nextInt();
	
	if(userInput == answer) {
		countCorrect += 1;
	}
	
	if(userInput != answer) {
		System.out.print("Try again: ");
		int userInputTwo = input.nextInt();
		if(userInputTwo != answer) {
			System.out.println("The correct answer is : " + answer);
		}
	}
	
}
System.out.print("You answered " + countCorrect + " questions out of 10 questions correctly!");

}
}

File: Javascript/DayOne/day_one.js
let prompt = require('prompt-sync')();

let count_correct = 0;
for(let count = 0; count < 10; count++) {
	let first_number = Math.floor(Math.random() * 100);
	let second_number = Math.floor(Math.random() * 100);
	let answer = first_number - second_number;
	

	let user_input = prompt("What is " + first_number + " minus " + second_number + " : ");

	if(user_input == answer) {
		countCorrect += 1;
	}
	
	if(user_input != answer) {
		let user_input2 = prompt("Try again: ");
		if(user_input2 != answer) {
			console.log("The correct answer is: " + answer);
		}
	}

}

console.log("You answered " + count_correct + " questions out of 10 questions correctly!");
